src/k8s-engine/wsl.ts

```diff
@@ -994,7 +1019,13 @@ export default class WSLBackend extends events.EventEmitter implements K8s.Kuber
               LOG_DIR:           logPath,
             });
             await this.writeFile('/etc/logrotate.d/k3s', rotateConf, 0o644);
-            this.runInit();
+            await this.runInit();
+            if (this.#currentContainerEngine !== ContainerEngine.MOBY) {
+              await this.writeFile(`/etc/init.d/buildkitd`, SERVICE_BUILDKITD_INIT, 0o755);
+              await this.writeFile(`/etc/conf.d/buildkitd`, SERVICE_BUILDKITD_CONF, 0o644);
+              await this.execCommand('/sbin/rc-update', '--update');
```

Comment:
This is not necessary, as `buildkitd` doesn't have dynamic dependencies.

Reply:
right